dbms/src/AggregateFunctions/AggregateFunctionAvgWeighted.h

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+#include <AggregateFunctions/AggregateFunctionAvg.h>
+
+namespace DB
+{
+template <typename T>
+struct AggregateFunctionAvgWeightedData
```

Comment:
I don't see any difference with `AggregateFunctionAvgData`. Why don't we use it?